covid_xprize/standard_predictor/xprize_predictor.py

```diff
@@ -94,13 +96,18 @@ def predict(self,
                 start_date_str: str,
                 end_date_str: str,
                 path_to_ips_file: str) -> pd.DataFrame:
+        # Load the npis into a DataFrame, handling regions
+        npis_df = self.load_original_data(path_to_ips_file)
+        return self.predict_from_df(start_date_str, end_date_str, npis_df)
```

Comment:
Load the NPIs into a DataFrame and call the `predict_from_df` api that deals with DataFrames.